MonoGame.Framework/Platform/SDL/SDLGamePlatform.cs

```diff
@@ -233,6 +238,46 @@ private void SdlRunLoop()
                                 break;
                         }
                         break;
+
+                    case Sdl.EventType.DropFile:
+                        if (ev.Drop.WindowId != _view.Id)
+                            break;
+
+                        if (!_view.IsFileDropHandled)
+                            break;
+
+                        unsafe
```

Comment:
You've got https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Platform/Utilities/InteropHelpers.cs#L16 to help you with converting IntPtr to string.